js/app.js

```diff
@@ -0,0 +1,67 @@
+//Section 1: Terms
+// 1. concatenation is combining two strings together. An example would be if you have two variables that you combine in a console.log as shown below:
+const pet = "dog";
+const name = "Mace";
+console.log("My " + pet + "'s " + "name is " + name);
+// 2. DRY means Don't Repeat Yourself. This aims to reduce repitition. Writing Loops for example can assist in writing DRY code.
+
+//Section 2
+const a = 4;
+const b = 53;
+const c = 57;
+const d = 16;
+const e = 'Kevin';
+
+console.log(a < b);
+console.log(c > d);
+console.log('Name' === 'Name');
+console.log(a > b < c);
+console.log(a === a < d);
+console.log(e === 'Kevin');
+console.log(48 != '48');
+
+//Section 3
+//Infinite Loop? 
+// This is an infinite loop because the statement will always be true. There is nothing to make it false.
+
+// Infinite Loop II
+// This is not an infinite loop because the main state of the variable runProgram is true however, the if the variable is pronounced false, the console.log will appear.
```

Comment:
a few things.  your initial statement is incorrect -- the loop is not infinite because it will not run more than once.  conceptually this is because we are trying to the value of runProgram to false in the code block itself, which means it would run only once. in reality it will throw an error because you're trying to change the value of a `const`.